react/SquareApp/Readme.md

```diff
@@ -0,0 +1,14 @@
+Component used to show an app status, can be used in a list of apps and connectors in the Home
+application.
+
+```
+import SquareApp from '.';
+
+<div className='u-flex' style={{padding: "1em", gap: "1em", backgroundColor: '#9DA2A6'}}>
```

Comment:
Pourquoi forcer un `backgroundColor: '#9DA2A6'` ?

Reply:
This is removed